docs/ug/cli.md

```diff
@@ -149,14 +155,14 @@ This flag overrides the `Ignore standalone config` field in the CSV config file.
 </box>
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### `--repos`, `-r`
+### `--repo`, `--repos`, `-r`
 
-**`--repos REPO_LOCATION`**: Specifies which repositories to analyze.
+**`--repo REPO_LOCATION`**: Specifies which repositories to analyze.
 * Parameter: `REPO_LOCATION` A list of URLs or the disk location of the git repositories to analyze, separated by spaces.
 * Alias: `-r`
 * Examples:
   * `--repos https://github.com/reposense/RepoSense.git`
-  * `--repos https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar`: analyzes the two specified repos (one remote, one local) and generates one report containing details of both.
+  * `--repo https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar`: analyzes the two specified repos (one remote, one local) and generates one report containing details of both.
```

Comment:
I think it makes sense to remove `--repo` since users would normally want to run RepoSense for multiple repos, although this should be done in another PR.

Reply:
I think my point was more that I don't think there's any benefit in specifying to users more than one option of the long flag. Like there's no need to give them the option of both `--repo` and `--repos`

Reply:
I am of the opinion that all usable flags should be documented, hence why I included both `--repo` and `--repos`.

Reply:
I see. Well I think I'll let the seniors decide on this since I think the rest of the PR is good to merge. @reposense/stage-1-active-reviewer

Reply:
I personally prefer not to have any 'hidden' features, especially something very easily mistyped like `repo` vs `repos`. Remove from documentation together with the actual flag removal since it might take some time.